### ISSUE TITLE
Merge dev branch with new meta data and sqlite outputs into prefect branch

### ISSUE THREAD

**Issue by bendnorman** [catalyst-cooperative/pudl#1234]
The prefect branch failed when trying to run the full etl because of a harvesting consistency error.  It doesn't make much sense to track down these small errors when the branch is 8 months out of date. 

The dev branch now contains new data and significant changes to our metadata and load steps. This is going to be a crazy merge that will require a lot of editing. 

This issue can be closed once dev is merged into prefect and the full etl can be run.

**Comment by bendnorman** [comment]
- [ ] Full ETL can run
- [ ] Passes unit tests
- [ ] Passes integration tests
- [ ] Passes validation tests

**Comment by bendnorman** [comment]
Integrated 0.5.0 release into the prefect branch. On to documentation and clean up!
